src/components/Form/Form.tsx

```diff
@@ -43,45 +47,61 @@ const Form = <T extends FormDetails>({
   hideCancelButton = false,
   ...props
 }: Props<T>) => {
+  const { t } = useTranslation();
   const initial = { form: props.defaults, errors: {} };
   const [isLoading, setIsLoading] = useState(!!asyncGetDefaults);
   const [state, dispatch] = useAutoSaveReducer<T>(formReducer, initial);
+  const [isEdited, setIsEdited] = useState(false);
```

Comment:
https://www.react-hook-form.com/api/useform/formstate/

use the in-built `isDirty` attribute instead.